sunpy/map/mapbase.py

```diff
@@ -1276,9 +1274,7 @@ def plot(self, gamma=None, annotate=True, axes=None, **imshow_args):
             axes.set_ylabel(ylabel)
 
 
-        cmap = deepcopy(self.cmap)
-        if gamma is not None:
-            cmap.set_gamma(gamma)
+        cmap = deepcopy(self.plot_settings['cmap'])
```

Comment:
I don't think we need the deepcopy here, the cmap should never be modified by the imshow command to my knowledge.